examples/sftda/01-spin_square.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env/python
+
+'''
+Test the spin_square calculation tool in sftda.tools_td.py.
+'''
+
+from pyscf import gto,dft,sftda,tdscf
+from pyscf.sftda import tools_td
+
+# ToDo : add the spin flip TDDFT parts.
+
+mol = gto.Mole()
+mol.verbose = 3
+mol.output = None
+mol.spin = 2
+mol.atom = 'O 0 0 2.07; O 0 0 0'
+mol.unit = 'B'
+mol.verbose = 6
+mol.basis = '6-31g'
+mol.build()
+
+mf = dft.UKS(mol)
+mf.xc = 'svwn'
+mf.kernel()
+
+#
+# 1. <S^2> for spin flip up TDA
+#
+mftd1 = sftda.TDA_SF(mf)
+mftd1.extype=0
+mftd1.nstates = 5
+mftd1.kernel()
+
+# extype serves for controling spin excitation type in the spin_square function.
```

Comment:
Please add more comments to this attribute, such as how the 0, 1, 2 corresponds to the spin flip cases.

Reply:
OK, I will add these comments to this attribute.